strings/index.js

```diff
@@ -37,7 +37,7 @@ export const howLong = (str) => {
   I expect this function to join the strings a and b together
 */
 export const join = (a, b) => {
-
+  return a+b; // I expected concat(a, b) to also work, but it didnt??
```

Comment:
Concat should work yeah but your implementation is slightly off.

From the [MDN Docs for string.concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat) it says the implementation for the function goes like this:

```js
const joinedString = stringA.concat(stringB);
```

However, using `.concat` is really only useful if you have a load of strings you want to concatenate together - so it wouldn't be my go to for this solution but it is correct.

Using + is also correct but it comes with 1 small caveat: using + as a way of joining strings can return an incorrect value if accidentally someone passes integers into the function.

```js
join(2, 3) === 5 // we expect this to return "23" because we are joining the string 2 and the string 3 together 
```
With that issue in mind, add another test to the test-suite for the join function
```js
expect(join(2, 3)).toEqual("23");
```
With your current implementation that test will fail. You can address this by checking if the variable is a string (you've already done this 😏) and if it isn't you can attempt to convert it a string (this is called 'casting') with the following:
```js
/*
  Note: the uppercase S - you are calling to initialise the base JS string class 
  with a number - if you remember much about class initialisation from your
  codecademy stuff
*/
const convertedString = String(203)
// "203"
```